### PR TITLE
Migrating store config to queries subdir as per Magedoc3976

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) moves the Store endpoint to the new queries subdir.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/reference/store-config.html

### CODE REVIEW

File: guides/v2.3/graphql/queries/store-config.md [hunk 1]
```diff
@@ -339,3 +225,118 @@ The [ProductInterface]({{ page.baseurl }}/graphql/reference/products.html#Produc
   }
 }
 ```
+
+### Extend configuration data
```
Comment (keharper) [inline guides/v2.3/graphql/queries/store-config.md]: Move this section to the end and change the heading to H2

File: guides/v2.3/graphql/queries/store-config.md [hunk 2]
```diff
@@ -339,3 +225,118 @@ The [ProductInterface]({{ page.baseurl }}/graphql/reference/products.html#Produc
...
+```
+
+You must also extend the type `storeConfig` within in the `etc/schema.graphqls` file, as shown below:
+
+```graphql
+type StoreConfig {
+    section_group_field : String  @doc(description: "Extended Config Data - section/group/field")
+}
+```
+
```
Comment (keharper) [inline guides/v2.3/graphql/queries/store-config.md]: Create a H2 called Output attributes. Bump the "Supported" headings down to H3.

Comment (erikmarr) [reply guides/v2.3/graphql/queries/store-config.md]: done

File: guides/v2.3/graphql/queries/store-config.md (hunk 1 above)
Comment (erikmarr) [reply guides/v2.3/graphql/queries/store-config.md]: done

**Comment by erikmarr** [comment]
running tests

**Pull request merged**